addons/sale_crm/models/crm_lead.py

```diff
@@ -60,3 +60,26 @@ def retrieve_sales_dashboard(self):
 
         res['invoiced']['target'] = self.env.user.target_sales_invoiced
         return res
+
+    def action_sale_quotations_new(self):
+        if not self.partner_id:
+            return self.env.ref("sale_crm.action_crm_quotation_partner").read()[0]
+        else:
+            return self.redirect_to_new_quotation()
+
+
+    def redirect_to_new_quotation(self):
```

Comment:
`action_new_quotation`. and remove one of the 2 blank line above method declaration ;)

Reply:
Hooo ! This is MY kind of review comments ! You thief ! :D